ShopAPI/ShopAPI/Controllers/CartController.cs

```diff
@@ -11,118 +11,17 @@ namespace ShopAPI.Controllers
 {
     [Route("api")]
     [ApiController]
-    public class ShopController : ControllerBase
+    public class CartController : ControllerBase
     {
         private readonly ICartService _cartService;
         private readonly IProductService _productService;
 
-        public ShopController(ICartService cartService, IProductService productService)
+        public CartController(ICartService cartService, IProductService productService)
         {
             _cartService = cartService;
             _productService = productService;
         }
```

Comment:
dont need the product service anymore since this is just a cart controller now

Reply:
Need both services in the cart controller because AddItemToCart uses both product and cart services